spec/src/modules/search.js

```diff
@@ -292,24 +291,6 @@ describe('ConstructorIO - Search', () => {
       });
     });
 
-    it('Should return a response from collection with a valid query and collection id', (done) => {
```

Comment:
why are we removing these collection tests? Are these no longer supported?

Reply:
Yea, the search endpoint no longer supports collection ids. It just returns an error:

`"Search within a collection is not supported."`